### DISCUSSION TITLE
Feature Request: display profile picture next to a mention

### DISCUSSION THREAD

**Discussion by GuillaumeDesforges** [community/community#36959] [Other Feature Feedback, Questions, & Ideas]
It can be hard to keep track of people's name, even more so for handles.

Some other discussion platforms like Basecamp add the profile picture next to mentions: https://basecamp.com/features/message-board

**Comment by ossanna16** [comment]
Hi there @GuillaumeDesforges and welcome to our community 😄. Thanks for the suggestion!
